### ISSUE TITLE
stop using https://w3c-ccg.github.io/ldp-bbs2020/context/v1

### ISSUE THREAD

**Issue by OR13** [mattrglobal/jsonld-signatures-bbs#101]
This context is still loaded when calling issue... instead, this one should be used:

https://w3id.org/security/bbs/v1

possibly after updating to make it match:

https://github.com/transmute-industries/vc.js/blob/master/packages/vc.js/src/__fixtures__/contexts/zkp-v1.json

**Comment by OR13** [comment]
hmm, can't fix the iri for the proof... without changes to this library... this is the best I can do:

```
{
  "@context": {
    "@version": 1.1,
    "id": "@id",
    "type": "@type",
    "BbsBlsSignature2020": {
      "@id": "https://w3c-ccg.github.io/ldp-bbs2020/context/v1#BbsBlsSignature2020",
      "@context": {
        "@version": 1.1,
        "@protected": true,
        "id": "@id",
        "type": "@type",
        "challenge": "https://w3id.org/security#challenge",
        "created": {
          "@id": "http://purl.org/dc/terms/created",
          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
        },
        "domain": "https://w3id.org/security#domain",
        "proofValue": "https://w3id.org/security#proofValue",
        "nonce": "https://w3id.org/security#nonce",
        "proofPurpose": {
          "@id": "https://w3id.org/security#proofPurpose",
          "@type": "@vocab",
          "@context": {
            "@version": 1.1,
            "@protected": true,
            "id": "@id",
            "type": "@type",
            "assertionMethod": {
              "@id": "https://w3id.org/security#assertionMethod",
              "@type": "@id",
              "@container": "@set"
            },
            "authentication": {
              "@id": "https://w3id.org/security#authenticationMethod",
              "@type": "@id",
              "@container": "@set"
            }
          }
        },
        "verificationMethod": {
          "@id": "https://w3id.org/security#verificationMethod",
          "@type": "@id"
        }
      }
    },
    "BbsBlsSignatureProof2020": {
      "@id": "https://w3c-ccg.github.io/ldp-bbs2020/context/v1#BbsBlsSignatureProof2020",
      "@context": {
        "@version": 1.1,
        "@protected": true,
        "id": "@id",
        "type": "@type",
        "challenge": "https://w3id.org/security#challenge",
        "created": {
          "@id": "http://purl.org/dc/terms/created",
          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
        },
        "domain": "https://w3id.org/security#domain",
        "nonce": "https://w3id.org/security#nonce",
        "proofPurpose": {
          "@id": "https://w3id.org/security#proofPurpose",
          "@type": "@vocab",
          "@context": {
            "@version": 1.1,
            "@protected": true,
            "id": "@id",
            "type": "@type",
            "assertionMethod": {
              "@id": "https://w3id.org/security#assertionMethod",
              "@type": "@id",
              "@container": "@set"
            },
            "authentication": {
              "@id": "https://w3id.org/security#authenticationMethod",
              "@type": "@id",
              "@container": "@set"
            }
          }
        },
        "proofValue": "https://w3id.org/security#proofValue",
        "verificationMethod": {
          "@id": "https://w3id.org/security#verificationMethod",
          "@type": "@id"
        }
      }
    },
    "Bls12381G2Key2020": "https://w3id.org/security#Bls12381G2Key2020"
  }
}
```

**Comment by tplooker** [comment]
Hey @OR13 we are currently fixing this for our next release, are you using the latest stable release of the library?

**Comment by tplooker** [comment]
Because the latest unstable release begins this migration to the security namespace, not complete yet however.

**Comment by OR13** [comment]
yes, unstable works... although looked at security-v3-unstable... I wonder if is not better to just ship a separate context file for every suite from now on, and they aggregate them in security-v3 later.... 

I think @kdenhartog was right about smaller context files... for the suite, what is the point of using security-v3-unstable over https://w3id.org/security/bbs/v1 if they define terms the same way? one of them includes a lot terms this suite will never use, the other does not...

**Comment by OR13** [comment]
I suggest you publish a stable version that fixes the IRIs asap... it will be a breaking change...the sooner you do it, the better.

**Comment by OR13** [comment]
This issue should be closed when the stable version does not have this stuff in it.

**Comment by kdenhartog** [comment]
I'm swamped at the moment, but if I catch some time to work on this I'll get it updated. I agree with your conclusion @OR13

**Comment by tplooker** [comment]
This issue will be addressed in the up and coming v0.8.0 release
